src/commands/add.js

```diff
@@ -59,6 +59,12 @@ Examples:
     process.exit(exitCodes.OK)
   }
 
+  // we don't (yet) support scoped npm packages, so bail
+  if (parameters.second.startsWith('@')) {
+    print.error(`Error: Ignite CLI doesn't support prefixed npm packages yet, sorry about that. Try it without the prefix.`)
```

Comment:
Can you change the message to call them scoped packages? https://docs.npmjs.com/getting-started/scoped-packages